Zend/tests/anon/006.phpt

```diff
@@ -10,6 +10,6 @@ namespace {
     var_dump ($hello);
 }
 ?>
---EXPECTF--
-object(class@%s)#1 (0) {
+--EXPECT--
+object(lone\@anonymous)#1 (0) {
```

Comment:
I would be happy if the "1st parent non-anonymous class" can be kept dumped, as from anonymous class it is often not easy to tell quickly what the anonymous class is about and the "1st parent non-anonymous class" was helpful, compare:

`DateTime@anonymous` vs. `class@anonymous` (or `Ns\@anonymous`) - https://3v4l.org/Bj2qg